apps/api/package.json

```diff
@@ -95,7 +96,10 @@
   },
   "devDependencies": {
     "@akashnetwork/dev-config": "*",
+    "@akashnetwork/docker": "*",
+    "@akashnetwork/releaser": "*",
     "@faker-js/faker": "^8.4.1",
+    "@release-it/conventional-changelog": "github:ygrishajev/conventional-changelog#feature/pre-release",
```

Comment:
Perhaps we can fork this one as well in the akash-network org

Reply:
Yea, sure